### PR TITLE
Fix miscounted unticketed events

### PR DESCRIPTION
**Pull request by dpanta94**
### 🎫 Ticket

[ET-2221]

### 🗒️ Description

In the event's admin list, having ET enabled, the ticketed and un-ticketed events can be miscounted for.

That happens because in the custom query the events with status auto-draft are not excluded.

Steps to reproduce:

- Fresh WP
- Install TEC
- Install ET
- Go to event's add new - exit without saving immediately
- Go back to event's admin list - you should see 1 event Untickted
- Repeat - the number should rise.

### 🎥 Artifacts 

image

### ✔️ Checklist
- [x] Ran `npm run changelog` to add changelog file(s). More info [here](https://docs.theeventscalendar.com/developer/git/changelogs/#process)
- [x] Code is covered by **NEW** `wpunit` or `integration` tests.
- [x] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [x] Are all the **required** tests passing?
- [x] Automated code review comments are addressed.
- [x] Have you added Artifacts?
- [x] Check the base branch for your PR.
- [x] Add your PR to the project board for the release.

[ET-2221]: https://stellarwp.atlassian.net/browse/ET-2221?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by rafsuntaskin — DISMISSED** [review]
Maybe exclude the `trash` as well as done in the original query: https://github.com/the-events-calendar/event-tickets/blob/fe14ac140c4ab8ba25d442c110a3ce5e59cf70a0/src/Tribe/Query.php#L203

While at it, we may also fix the [ticketed count](https://github.com/the-events-calendar/event-tickets/blob/b173703378d545f4036160690a6f8f679ac029ef/src/Tickets/Flexible_Tickets/Series_Passes/Queries.php#L95) query as well.

Need to add tests under `ft_integration` suite as this issue is coming from Flexible tickets. We should already have some tests for the counts in the existing tests.

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 ticketed and 6 unticketed posts, 2 invalid post status in each case__0.snapshot.html
Comment (rafsuntaskin) [inline tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 ticketed and 6 unticketed posts, 2 invalid post status in each case__0.snapshot.html:1]: Is this correct? It shows no changes.

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 unticketed posts, 3 valid and 2 invalid post status__0.snapshot.html
Comment (rafsuntaskin) [inline tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 unticketed posts, 3 valid and 2 invalid post status__0.snapshot.html:1]: Is this correct? It shows no changes.

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 ticketed and 6 unticketed posts, 2 invalid post status in each case__0.snapshot.html
Comment (dpanta94) [reply tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 ticketed and 6 unticketed posts, 2 invalid post status in each case__0.snapshot.html:1]: yes!

File: tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 unticketed posts, 3 valid and 2 invalid post status__0.snapshot.html
Comment (dpanta94) [reply tests/ft_integration/TEC/Tickets/Flexible_Tickets/Series_Passes/__snapshots__/Series_PassesTest__should_correctly_filter_the_ticketed_unticketed_labels__5 unticketed posts, 3 valid and 2 invalid post status__0.snapshot.html:1]: yes!

**Review by bordoni — APPROVED** [review]

**Pull request merged**